crates/unftp-sbe-fs/README.md

```diff
@@ -1,6 +1,70 @@
 # unftp-sbe-fs
 
-An storage back-end for [libunftp](https://github.com/bolcom/libunftp) that let you store files on the local filesystem.
+This unftp-sbe-fs crate allows you to use a regular Filesystem with
+[libunftp](https://github.com/bolcom/libunftp) and work like a regular
+FTP server.
+
+## Getting started
+
+If you've got Rust and cargo installed, create your project with
+
+```sh
+cargo new myftp
+```
+
+Add the libunftp and tokio crates to your project's dependencies in `Cargo.toml`.
+
+```toml
+[dependencies]
+libunftp = "0.17.4"
+unftp-sbe-fs = "0.1"
+tokio = { version = "1", features = ["full"] }
+```
+
+Now you're ready to develop your server!
+Add the following to `src/main.rs`:
+
+```rust
+use unftp_sbe_fs::ServerExt;
+
+#[tokio::main]
+pub async fn main() {
```

Comment:
I'm adding the examples to the cargo docs too, think it also needs something: https://docs.rs/unftp-sbe-fs/0.1.0/unftp_sbe_fs/

Reply:
Added, please check.